### DISCUSSION TITLE
Simplest way for add captha to user registration

### DISCUSSION THREAD

**Discussion by ludimassa** [Oksydan/falcon#336] [Ideas]
Hi everyone, anyone know the best and simplest way for add captha to the user registration page?

**Comment by Oksydan** [comment]
Hi @ludimassa,

The easiest way should be to use a module. Unfortunately, I can't provide a ready-made solution because I don't have one.
There is an open source project https://github.com/nenes25/eicaptcha - but I have not tested it.
